### PR TITLE
Improve CSS rules of zoom buttons on live location

### PR DESCRIPTION
**Pull request by luixxiul**
This PR:

- Use flex to stop unnecessary margin-top from appearing. Canceling the margin `&:first-child` with `margin-top: 0` would be redundant.
- Remove ButtonResetDefault. Here it resets nothing, and also would possibly hide an issue of the declarations related to cascading.
- Ensure that unexpected regressions will never happen on the buttons and their icons. Rules can be extracted when needed. Until then there is not a point of exposing elements to possible regressions, which could easily happen on the current architecture where every top level block is treated equally without consideration about cascading.

|Before|After|
|---------|------|
|before|after|

type: task

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: res/css/components/views/location/_ZoomButtons.scss [hunk 1]
```diff
@@ -16,30 +16,30 @@ limitations under the License.
...
-    bottom: $spacing-32;
-    right: $spacing-24;
-}
-
-.mx_ZoomButtons_button {
-    @mixin ButtonResetDefault;
-
-    margin-top: $spacing-8;
-    border-radius: 4px;
+    inset-block-end: $spacing-32;
```
Comment (weeman1337) [inline res/css/components/views/location/_ZoomButtons.scss:19]: Can we stick to `bottom` and `right` here? There is no text where buttons should be be aligned to depending on the text direction. Other elements on the view are using `top`, `left`… as well. Means if a user has a different text direction the layout may break.

**Review by weeman1337 — CHANGES_REQUESTED** [review]
Thanks again for your contribution @luixxiul :+1: 

Looks good except one tiny point. Please check my comment.

File: res/css/components/views/location/_ZoomButtons.scss (hunk 1 above)
Comment (luixxiul) [reply res/css/components/views/location/_ZoomButtons.scss:19]: That makes sense. They could be changed at the same time in order to make the whole UI compatible with RTL layout.

Comment (luixxiul) [reply res/css/components/views/location/_ZoomButtons.scss]: The commit was dropped.

**Comment by weeman1337** [comment]
@luixxiul can you once again merge `develop`?

**Comment by luixxiul** [comment]
@weeman1337 Done!

**Review by weeman1337 — APPROVED** [review]

**Comment by weeman1337** [comment]
Tested, works; LGTM now :+1:

**Comment by luixxiul** [comment]
@weeman1337 Thanks for checking!

**Pull request merged**